src/cs3panel.tsx

```diff
@@ -89,18 +93,67 @@ export class Cs3HeaderWidget extends ReactWidget {
     }
 }
 
+export const Bottom = (props: BottomProps): JSX.Element => {
+
+    const [text, setText] = useState(props.message);
+
+    const onPathOrFileChanged = async (path: string) => {
+        const showHidden: boolean = await props.db.fetch('showHidden') as boolean;
+        if (showHidden == undefined || !showHidden) {
+            const result: Contents.IModel = await requestAPI('/api/contents/' + path, {method: 'get'});
```

Comment:
So you do an extra call (repeated), just to count the current hidden files? Isn't there a way to get this info when we receive the current folder listing? (didn't you do that drive.ts already?)
I would expect that showing/hiding hidden files would result in 0 calls to the server. Ever.

Reply:
![hidden-hide](https://user-images.githubusercontent.com/61541089/133151614-924582da-02b8-45ba-a39f-7db63d8cda06.JPG)
![hidden-show](https://user-images.githubusercontent.com/61541089/133151617-8844d5c9-8009-4284-8515-93027a703d62.JPG)